packages/composer.rb

```diff
@@ -25,10 +25,10 @@ def self.install
     system "curl -Ls -o composer https://github.com/composer/composer/releases/download/#{version}/composer.phar"
     abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read('composer') ) == '0a060e8caf1d5cde55c4562fd68369b08cf231b8fd30da9c8f994b111557ebe9'
     system "install -Dm755 composer #{CREW_DEST_PREFIX}/bin/composer"
-    FileUtils.ln_sf "#{CREW_PREFIX}/.config", "#{HOME}/.config"
   end
 
   def self.postinstall
+    FileUtils.ln_sf "#{CREW_PREFIX}/.config", "#{HOME}/.config"
```

Comment:
Why does this line need to move?

Reply:
Otherwise I have a ~/.config/.config/.config/.config/.config .... each time I upgrade or reinstall composer

Reply:
I guess I was asking why it was moved from `install` to the `postinstall` section.

Reply:
I understood it that way indeed. Answer is the same : if it is placed in the install section, I have recursive symbolic links which does not occure if it is placed in postinstall.... Absolutely no idea why....